tests/coalaHTMLTest.py

```diff
@@ -40,10 +54,14 @@ def test_output_file(self):
                           "-c", os.devnull,
                           "-b", "LineCountBear",
                           "-f", re.escape(filename),
-                          "--noupdate",
-                          "--nolaunch")
+                          "--noupdate")
 
             with open(self.result_file, 'r') as fp:
                 noupdate_file = fp.read()
 
         self.assertEqual(update_file, noupdate_file)
+        # We `chdir` before spawning the server
+        shutil.rmtree('../coalahtmlapp', ignore_errors=True)
```

Comment:
It doesn't look like you're doing chdir here, also why not put all the cleanup into the tearDown?